spec/lib/http_spec.rb

```diff
@@ -307,6 +307,15 @@
       end
     end
 
+    context "specifying per operation timeouts as frozen hash" do
+      let(:frozen_options) { {:read => 123}.freeze }
+      subject(:client) { HTTP.timeout(frozen_options) }
+
+      it "does not raise an error" do
+        expect { client }.not_to raise_error(FrozenError)
```

Comment:
Please don't specify error class. This code simply *should not raise*. It's generally recommended to avoid specify error class on `not_to raise_error`.

```suggestion
        expect { client }.not_to raise_error
```

Reply:
Done 👍 and rebased